admin/load.php

```diff
@@ -339,6 +339,84 @@ function perflab_get_module_data( $module_file ) {
 	return $module_data;
 }
 
+/**
+ * Initialise Admin Pointer
+ *
+ * Handles the bootstrapping of the admin pointer.
+ * Mainly jQuery code that is self-initialising.
+ *
+ * @param string $hook_suffix The current admin page.
+ * @since 1.0.0
```

Comment:
Numbers here should be `1.0.0-beta.1` ?

Reply:
same question for all version numbers below

Reply:
@mitogh @adamsilverstein The versions in the codebase should still always be the "main" version that it applies to. This is similar in WP core and also documented in https://github.com/WordPress/performance/blob/trunk/docs/Releasing-the-plugin.md#update-the-version-number